packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -256,6 +290,86 @@ export class ARRenderer extends EventDispatcher<
     this.dispatchEvent({type: 'status', status: ARStatus.SESSION_STARTED});
   }
 
+  private setupControllers() {
+    this.controller1 = this.threeRenderer.xr.getController(0);
+    this.controller1.addEventListener(
+        'selectstart',
+        (e: XRControllerEvent) => this.onControllerSelectStart(e));
+    this.controller1.addEventListener(
+        'selectend', (e: XRControllerEvent) => this.onControllerSelectEnd(e));
+
+    this.controller2 = this.threeRenderer.xr.getController(1);
+    this.controller2.addEventListener(
+        'selectstart',
+        (e: XRControllerEvent) => this.onControllerSelectStart(e));
+    this.controller2.addEventListener(
+        'selectend', (e: XRControllerEvent) => this.onControllerSelectEnd(e));
+
+    // const controllerModelFactory = new XRControllerModelFactory();
+
+    // this.controllerGrip1 = this.threeRenderer.xr.getControllerGrip(0);
+    // this.controllerGrip1.add(
+    //     controllerModelFactory.createControllerModel(this.controllerGrip1));
+
+    // this.controllerGrip2 = this.threeRenderer.xr.getControllerGrip(1);
+    // this.controllerGrip2.add(
+    //     controllerModelFactory.createControllerModel(this.controllerGrip2));
+
+    const line = new Line(lineGeometry);
+    line.name = 'line';
+    line.scale.z = 5;
```

Comment:
What is this 5 constant? maybe add a comment for this one

Reply:
Yeah, or even better, a named constant. Thanks.